### PR TITLE
Webhooks guide security

### PR DESCRIPTION
**Pull request by FlowFlorent**
### Description
- Adds Message verification section
- Remove mentions of svix
- Adds list of events

### CODE REVIEW

File: docs/pages/guides/webhooks.mdx [hunk 1]
```diff
@@ -54,18 +58,110 @@ which events make the most sense. The Event Catalog provides a list of events
 available for subscription, along with their schema.
 
 Events available for use include:
+
 - UserEntered/UserLeft
 - StorageUpdated
 - RoomCreated/RoomDeleted
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:64]: @FlowFlorent do we really support those events?

File: docs/pages/guides/webhooks.mdx [hunk 2]
```diff
@@ -54,18 +58,110 @@ which events make the most sense. The Event Catalog provides a list of events
...
+		connectionId: 4
+		userId: '1234sdgs'
+		userInfo: null
+		enteredAt: '2021-10-06T01:45:56.558Z'
+		numActiveUsers: 8
+	}
+}
+```
+
+#### UserLeft
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:99]: Could we have a short description of what it is in human language?

File: docs/pages/guides/webhooks.mdx [hunk 3]
```diff
@@ -119,54 +209,76 @@ If your service is unreachable, retries will automatically be attempted. If your
...
+- Immediately
+- 5 seconds
+- 5 minutes
+- 30 minutes
+- 2 hours
+- 5 hours
+- 10 hours
+- 10 hours (in addition to the previous)
+
+If an endpoint is removed or disabled delivery attempts to the endpoint will be
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:232]:
```suggestion
If an endpoint is removed or disabled, delivery attempts to the endpoint will be
```

File: docs/pages/guides/webhooks.mdx [hunk 4]
```diff
@@ -119,54 +209,76 @@ If your service is unreachable, retries will automatically be attempted. If your
...
+// Need to base64 decode the secret
+const secretBytes = new Buffer(secret.split("_")[1], "base64");
+// This is the signature you will compare against the signature header
+const signature = crypto
+  .createHmac("sha256", secretBytes)
+  .update(signedContent)
+  .digest("base64");
+```
+
+#### 2. Validate the signature to the signature header
```
Comment (FlowFlorent) [inline docs/pages/guides/webhooks.mdx:277]:
```suggestion
#### 2. Validate the signature
```

File: docs/pages/guides/webhooks.mdx (hunk 2 above)
Comment (FlowFlorent) [reply docs/pages/guides/webhooks.mdx:106]: good catch, added

File: docs/pages/guides/webhooks.mdx [hunk 5]
```diff
@@ -54,18 +58,125 @@ which events make the most sense. The Event Catalog provides a list of events
...
+More events will come later such as:
+
 - RoomCreated/RoomDeleted
 - MaxConnectionsReached
 
+#### UserEntered
+
+A user enters a room when they connect to a room. This event is triggered when a
+user enters a room. The number of active users is the number of users after the
+user has entered.
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:74]:
```suggestion
A user enters a room when they connect to a room, which is when this event is triggered. The number of active users is the number of users in the room after the
user has entered.
```

File: docs/pages/guides/webhooks.mdx [hunk 6]
```diff
@@ -54,18 +58,125 @@ which events make the most sense. The Event Catalog provides a list of events
...
+    numActiveUsers: 8,
+  },
+};
+```
+
+#### UserLeft
+
+A user leaves a room when they disconnect from a room. This event is triggered
+when a user leaves a room. The number of active users is the number of users
+after the user has left.
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:110]:
```suggestion
A user leaves a room when they disconnect from a room, which is when this event is triggered. The number of active users is the number of users in the room 
after the user has left.
```

File: docs/pages/guides/webhooks.mdx [hunk 7]
```diff
@@ -54,18 +58,125 @@ which events make the most sense. The Event Catalog provides a list of events
...
+    numActiveUsers: 7,
+  },
+};
+```
+
+#### StorageUpdated
+
+Storage is updated when a user writes to storage. This event is throttled and as
+such may not be triggered for every write. The `updatedAt` field is the time of
+the last write.
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:148]:
```suggestion
Storage is updated when a user writes to storage. This event is throttled and, as
such, may not be triggered for every write. The `updatedAt` field is the time of
the last write.
```

File: docs/pages/guides/webhooks.mdx [hunk 8]
```diff
@@ -54,18 +58,123 @@ which events make the most sense. The Event Catalog provides a list of events
...
+    enteredAt: "2021-10-06T01:45:56.558Z",
+    numActiveUsers: 8,
+  },
+};
+```
+
+#### UserLeft
+
+A user leaves a room when they disconnect from a room, which is when this event is triggered. The number of active users is the number of users in the room 
+after the user has left.
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:108]:
```suggestion
A user leaves a room when they disconnect from a room, which is when this event is triggered. The `numActiveUsers` field represents the number of users in the room 
after the user has left.
```

File: docs/pages/guides/webhooks.mdx [hunk 9]
```diff
@@ -54,18 +58,123 @@ which events make the most sense. The Event Catalog provides a list of events
...
+
+More events will come later such as:
+
 - RoomCreated/RoomDeleted
 - MaxConnectionsReached
 
+#### UserEntered
+
+A user enters a room when they connect to a room, which is when this event is triggered. The number of active users is the number of users in the room after the
+user has entered.
```
Comment (adigau) [inline docs/pages/guides/webhooks.mdx:73]:
```suggestion
A user enters a room when they connect to a room, which is when this event is triggered. The `numActiveUsers` field represents the number of users in the room 
after the user has entered.
```

**Pull request merged**